backend/data_handling/data_processing.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
```

Comment:
Your in-line comments have really improved-- your comments in this file are great!